src/__tests__/autocapture.js

```diff
@@ -897,12 +897,6 @@ describe('Autocapture system', () => {
 
             jest.spyOn(autocapture, '_addDomEventHandlers')
         })
-
-        it('should check whether to load the editor', () => {
-            given.subject()
-
-            expect(given.lib.toolbar.maybeLoadEditor).toHaveBeenCalled()
-        })
```

Comment:
Why not keep the test but move it to a different place (since the function still should be called, just elsewhere)?

Reply:
I tried, but I couldn't figure out how to make it happen in this environment. :(